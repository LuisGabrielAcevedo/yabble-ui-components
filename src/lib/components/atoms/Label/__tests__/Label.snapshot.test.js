import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  storyKindRegex: 'Yabble/Atoms/Label',
})
