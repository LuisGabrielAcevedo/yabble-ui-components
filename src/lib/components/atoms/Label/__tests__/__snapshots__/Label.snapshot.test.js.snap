// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Yabble/Atoms/Label Default Label 1`] = `
<div
  className="label-container"
>
  <label
    className="label "
    htmlFor=""
    style={Object {}}
  >
    First name
  </label>
</div>
`;
